index.md

```diff
@@ -23,6 +32,32 @@ Or get in touch with any of the current organizing members:
 * [Andrew Walter](https://www.khoury.northeastern.edu/people/andrew-thomas-walter/),
 * [Ming-Ho Yee](https://www.khoury.northeastern.edu/people/ming-ho-yee/).
 
+### Useful Resources
+
+There are many problems that pop up again and again in front of a Khoury PhD student.
+KGSA tries to coordinate a wealth of common knowledge collected by many generations of the students.
+We suggest trying the following resources to find answers to your questions:
+
+* [Khoury PhD Pragrams page on Canvas][notebook-canvas]
+
+* [PhD Hub on Confluence][confluence]
+
+* [Crowd-sourced Google Doc][notebook-gdoc]
+
+Note that the first resource is administred by the College, whereas the other two
+are crowd-sourced by the students, and if you have ideas on what to add there,
```

Comment:
Does "there" only refer to Canvas? For the other two we probably don't want people to go through us.

Reply:
I didn't mean to propose Canvas updates, but to the other two places. But now I see it doesn't make much sense since everyone can commit there. I will remove this I guess.